packages/core/src/components/BulkActions/BulkActions.tsx

```diff
@@ -122,6 +126,11 @@ export const HvBulkActions = (props: HvBulkActionsProps) => {
   const anySelected = numSelected > 0;
   const isSemantic = semantic && anySelected;
 
+  const dividerColor =
+    activeTheme?.colors?.modes[selectedMode].type === "dark"
+      ? theme.colors.secondary
+      : theme.colors.atmo4;
```

Comment:
We shouldn't be doing this. This should be solved with the color palette (as we've always done), otherwise we're opening the door to maintainability and customisation issues.
If we need a new color for this, let's add it. Otherwise, we could keep `atmo4`, which is the _specs_ we have been following (zeroheight dawn)